pkg/hubble/cell/cell.go

```diff
@@ -62,6 +70,20 @@ var Core = cell.Group(
 	cell.Config(defaultConfig),
 )
 
+// ConfigProviders provides configuration objects for Hubble components.
+// This group creates and provides configuration structs by combining
+// different configuration sources (hubble config, TLS config, etc.).
+var ConfigProviders = cell.Group(
+	// Provide HubbleConfig struct for peer service and other components
+	cell.Provide(func(cfg config, tlsCfg certloaderConfig) *peercell.HubbleConfig {
```

Comment:
nitpicking: don't expose this outside of the Hubble Cell

```suggestion
	cell.ProvidePrivate(func(cfg config, tlsCfg certloaderConfig) *peercell.HubbleConfig {
```